src/ripple/protocol/jss.h

```diff
@@ -639,6 +645,8 @@ JSS(validator_sites);         // out: ValidatorSites
 JSS(value);                   // out: STAmount
 JSS(version);                 // out: RPCVersion
 JSS(vetoed);                  // out: AmendmentTableImpl
+JSS(vol_a);                   // out: BookChanges
+JSS(vol_b);                   // out: BookChanges
```

Comment:
Can we come up with a better name for this? I'd rather spell out `volume` at least.